docs/reference/api/security/TLS.md

```diff
@@ -41,6 +41,26 @@ Then create a file named `mbed_app.json` at the root of your application with th
 }
 ```
 
+### Mbed TLS platform context
+
+Some hardware accelerators require initialization, regardless of the specific cryptography engine. For this, we introduced `mbedtls_platform_setup()` and `mbedtls_platform_terminate()`.
+
+As the [examples](#mbed-tls-examples) show, you *must* call the `mbedtls_platform_setup()` function before you call any Mbed TLS API. After using the Mbed TLS API, you *must* call `mbedtls_platform_teardown()` to terminate the platform hardware driver. We suggest you set the platform context parameter for these functions as `NULL`.
```

Comment:
I think, that the advice about passing NULL to the functions may be a little confusing. I am not sure if it is possible to explain it briefly enough so that it suits this document, but I'd definitely try doing it. Otherwise I would consider omitting this advice and leave it for the detailed documentation.

Reply:
That detailed documentation. This should be the detailed documentation, but again, not too detailed, as we don't want to confuse.
What is confusing?

Reply:
If I first saw such documentation I would have some questions in my mind, such as:

* If this function is so important (one *must* call it), why is its argument irrelevant?
* If I am to pass in NULL, why does the function take a parameter at all?
* What are the cases, in which we pass an actual object?
* If I want to know the details, where do I look? (OK this may be obviously the API reference, but we may put a link here, so that it is conveniently accessible)

To make myself clear - I think the way you put it is enough to use the API correctly, so I think it is correct itself. I am just trying to foresee issues, that new user may have.

Reply:
Briefly here: https://os-doc-builder.test.mbed.com/docs/development/apis/tls.html